app/views/shared/_notifications.html.erb

```diff
@@ -45,9 +45,10 @@
     <p>File exceedes feature limit <button class="btn little white source" data-source="drop-shapefile">More info</button></p>
   </div>
   <div id="notification-glad-data-alert" data-type="warning">
-    <p>GLAD alerts are currently on hold as we migrate the system to Google Earth Engine and adjust to recent changes made by Landsat.<br>
-    Check out our <a href="https://groups.google.com/forum/#!forum/globalforestwatch" class="text -color-2 -underline" target="_blank">Discussion Forum</a>
-    for the latest details or with any questions.</p>
+    <p>
+      GLAD alerts are currently being updated for Peru only - other areas are on hold (as of March 10) as we work to<br>
+      migrate the system to Google Earth Engine and adjust to recent changes made by Landsat. Check out our <a href="https://groups.google.com/forum/#!forum/globalforestwatch" target="_blank">Discussion Forum</a><br> for the latest details or with any questions.
```

Comment:
I think this is a must to read https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/ so I think we could add the `rel="noopener noreferrer"` to the target blank hrefs